test/cli/main.js

```diff
@@ -71,7 +71,8 @@ QUnit.module( "CLI Main", () => {
 		try {
 			await execute( "qunit syntax-error/test.js" );
 		} catch ( e ) {
-			assert.true( e.stdout.includes( "not ok 1 syntax-error/test.js > Failed to load the test file with error:" ) );
+			assert.true( e.stdout.includes( "not ok 1 global failure" ) );
```

Comment:
FWIW I've found
```js
assert.true(x.includes(y), x)
```
to be helpful (to add the "actual" as a diagnostic to inspect on failure)

Reply:
Ack. I saw that pattern in a few places. I also saw the `pushResult({ result: str.includes(), actual: str })` pattern in a few other places in this file which I think renders in a slightly less confusing way. We could pick one of those - or define a local `assert.includes` util to start with - and use that consistently for all similar needs in this file. (Also happy to take a patch for that after this lands 🙂 ).